===== Patch instructions generated by SAVER =====
[+] { Delete: if (true) free(n$9) at 12 (line 679, column 3) }
===== Actual patch in diff format =====
@@ -675,8 +675,6 @@ main (int argc,
  out:
 	server_free (server);
 
-	if (socket_name)
-		free (socket_name);
 	if (socket_base) {
 		remove (socket_base);
 		free (socket_base);
